core/commands/files/files.go

```diff
@@ -90,19 +109,58 @@ var FilesStatCmd = &cmds.Command{
 	},
 	Marshalers: cmds.MarshalerMap{
 		cmds.Text: func(res cmds.Response) (io.Reader, error) {
+
 			out := res.Output().(*Object)
 			buf := new(bytes.Buffer)
-			fmt.Fprintln(buf, out.Hash)
-			fmt.Fprintf(buf, "Size: %d\n", out.Size)
-			fmt.Fprintf(buf, "CumulativeSize: %d\n", out.CumulativeSize)
-			fmt.Fprintf(buf, "ChildBlocks: %d\n", out.Blocks)
-			fmt.Fprintf(buf, "Type: %s\n", out.Type)
+
+			s, _ := statGetFormatOptions(res.Request())
+			s = strings.Replace(s, "<hash>", out.Hash, -1)
+			s = strings.Replace(s, "<size>", fmt.Sprintf("%d", out.Size), -1)
+			s = strings.Replace(s, "<cumulsize>", fmt.Sprintf("%d", out.CumulativeSize), -1)
+			s = strings.Replace(s, "<childs>", fmt.Sprintf("%d", out.Blocks), -1)
+			s = strings.Replace(s, "<type>", out.Type, -1)
+
+			fmt.Fprintln(buf, s)
 			return buf, nil
 		},
 	},
 	Type: Object{},
 }
 
+func statGetFormatOptions(req cmds.Request) (string, error) {
+
+	hash, _, _ := req.Option("hash").Bool()
+	size, _, _ := req.Option("size").Bool()
+	formatSpecified := req.Option("format").Found()
+
+	count := 0
+	if hash {
+		count++
+	}
+	if size {
+		count++
+	}
+	if formatSpecified {
+		count++
+	}
+	if count > 1 {
+		return "", formatError
+	}
```

Comment:
i think this works:
```go
func moreThanOne(a,b,c bool) bool {
	return !((a == b == c) && !(a && b && c)) && (a || b || c)
}
```

it can probably be reduced...